<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMSDOS</class>
 <widget class="QWidget" name="FormMSDOS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>25</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabelX" name="labelEntryPoint">
     <property name="minimumSize">
      <size>
       <width>78</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>78</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string notr="true">EntryPoint:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEditX" name="lineEditEntryPoint">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButtonX" name="pushButtonEntryPoint">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string notr="true">&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>327</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButtonX" name="pushButtonOverlay">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Overlay</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButtonX" name="pushButtonDosHeader">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Dos Header</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPushButtonX</class>
   <extends>QPushButton</extends>
   <header>qpushbuttonx.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEditX</class>
   <extends>QLineEdit</extends>
   <header>qlineeditx.h</header>
  </customwidget>
  <customwidget>
   <class>QLabelX</class>
   <extends>QLabel</extends>
   <header>qlabelx.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
